com.airtame.Client.json

```diff
@@ -1,13 +1,16 @@
 {
   "app-id": "com.airtame.Client",
   "runtime": "org.freedesktop.Platform",
-  "runtime-version": "19.08",
+  "runtime-version": "21.08",
   "sdk": "org.freedesktop.Sdk",
-  "base": "io.atom.electron.BaseApp",
-  "base-version": "19.08",
+  "base": "org.electronjs.Electron2.BaseApp",
+  "base-version": "21.08",
   "command": "airtame",
+  "separate-locales": false,
   "finish-args": [
     "--share=ipc",
+    "--device=dri",
+    "--socket=pulseaudio",
```

Comment:
Why are these needed now? 🤔

Reply:
If it uses Electron it almost always needs access to dri for rendering (because Electron itself needs it).

```
info: [core-streamer]: [Streamer] Creating streamer 0x2a11af575d20 [version: v2.0.0]
libGL error: MESA-LOADER: failed to retrieve device information
libGL error: MESA-LOADER: failed to open amdgpu: /usr/lib/x86_64-linux-gnu/GL/default/lib/dri/amdgpu_dri.so: cannot open shared object file: No such file or directory (search paths /usr/lib/x86_64-linux-gnu/GL/default/lib/dri, suffix _dri)
libGL error: failed to load driver: amdgpu
[46:0809/083333.564331:ERROR:buffer_manager.cc(488)] [.DisplayCompositor]GL ERROR :GL_INVALID_OPERATION : glBufferData: <- error from pr
```

No idea about pulseaudio but in the logs I see that it trying to some audio capturing etc.

```
info: [core-streamer]: [audio_capture_rtaudio] Init called
info: [core-streamer]: [audio_capture_rtaudio] Found 1 audio devices: 
info: [core-streamer]: [audio_capture_rtaudio] device 0, name: PulseAudio, status: available, out_channels: 2, in_channels: 2
```